snapcraft/project/_project_options.py

```diff
@@ -131,19 +131,22 @@ def _get_platform_architecture():
     return architecture
 
 
-class ProjectOptions:
-    @property
-    def use_geoip(self):
-        return self.__use_geoip
+def _get_local_plugins_dir(project_dir: str, parts_dir: str) -> str:
+    deprecated_plugins_dir = os.path.join(parts_dir, "plugins")
+    if os.path.exists(deprecated_plugins_dir):
+        handle_deprecation_notice("dn2")
+        local_plugins_dir = deprecated_plugins_dir
+    else:
+        local_plugins_dir = os.path.join(project_dir, "snap", "plugins")
 
-    @property
-    def parallel_builds(self):
-        return self.__parallel_builds
+    return local_plugins_dir
 
+
+class ProjectOptions:
     @property
     def parallel_build_count(self):
         build_count = 1
-        if self.__parallel_builds:
+        if self.parallel_builds:
```

Comment:
This will probably need to be self._parallel_builds now, no?

Reply:
Ah, darn, I missed the fact that this property existed. No matter, it's fine either way.